src/gui/properties/speedplotview.cpp

```diff
@@ -38,10 +38,10 @@ SpeedPlotView::SpeedPlotView(QWidget *parent)
     , m_data5Min(MIN5_BUF_SIZE)
     , m_data30Min(MIN30_BUF_SIZE)
     , m_data6Hour(HOUR6_BUF_SIZE)
+    , m_averager30Min(divider30Min,m_data30Min)
+    , m_averager6Hour(divider6Hour,m_data6Hour)
```

Comment:
Need space beetwen the 2 parameter eg `m_averager6Hour(divider6Hour, m_data6Hour)`

Reply:
fixed